static_src/stores/service_instance_store.js

```diff
@@ -217,7 +217,8 @@ class ServiceInstanceStore extends BaseStore {
         const instance = this.get(binding.service_instance_guid);
         if (!instance) break; // TODO throw error
         const updatedInstance = Object.assign({}, instance, {
-          changing: false
+          changing: false,
+          error: false
```

Comment:
hmm when i think of `error` i expect like a string or an object not a `boolean`. maybe change this to `hasError` or something similar.